### DISCUSSION TITLE
use javascriptexecutor

### DISCUSSION THREAD

**Discussion by prasant-sethi** [webdriverio/webdriverio#7871] [Q&A]
Hi- I have been trying to figure out a way to use the execute js command in wdio but not able to. Here is an example code - 

const title = await browser.execute('return document.title');

This doesn't work. throws error:
Evaluation failed: Error: eval is disabled.

**Comment by christian-bromann** [comment]
Can you file an issue for this? It got introduced with the latest Chrome release.

**Reply by prasant-sethi** [reply]
Thank you. Opened an issue #7872

**Reply by christian-bromann** [reply]
Sorry, I initially thought it was a bug in WebdriverIO but turns out to be a problem with the page you are testing.

**Reply by prasant-sethi** [reply]
After your last response, I tested on safari and firefox. It works.
